### ISSUE TITLE
PIE pronouns WE and YOU_PL

### ISSUE THREAD

**Issue by RolandPooth** [lingdb/CoBL#269]
YOU_PL
Many of the forms do not belong to this cognate class - I presume the ones beginning with t- This class semes to need a thorough revision - it may need some days to run through all the etymological dictionaries. I suggest to advice the authors to revise their cognacy judgments. Personally, I would only include forms directly and securely continuing *(h₁)iú(:)(s) and *u̯ó- here. But, as for the problem of reconstruction of the PIE pronouns and possible mergers, please cf. below:

WE cognate class 158
PIE *u̯éi ~ *u̯éi̯es (as if it was a plural of an i-stem, cf. tréi̯es); some forms go back to a different stem *(h₁)més. There is even a third stem *nó-. PIE is thus likely to be of the type of languages who exhibit a first person plural inclusive versus exclusive distinction, which then collapsed. This cognate class 158 is full of flaws and should be revised, double-checked, or even excluded. The reconstruction of the PIE word for 'WE' is not finally solved and proposing a single cognate class, as it stands, can only lead to mistakes (Pooth).

**Comment by CasFre** [comment]
As far as WE cognate set 158 is concerned:
Normally, our policy is to treat suppletive stems together, cf. https://github.com/lingdb/CoBL/issues/208.
However, if you think that (1) the inclusive - exclusive distinction is reasonably well to establish and to defend and (2) the formal and functional differences between the three stems are too big to still call it mere suppletion, then you could split this cognate set up into two or three, one for each stem + meaning and cross-reference between them.
If lexemes from a single branch then still show suppletion between two or three of these stems, then you could assign these individual lexemes to the respective cognate classes.
Maybe that should reflect the cladistics best.

**Comment by RolandPooth** [comment]
Personally, I think that *u̯éi ~ *u̯éi̯es was inclusive, whereas *(h₁)més was exclusive, but people have not thought about this until these days ... There is also a problem with *u̯ó-. "Nom. 1st Dual" is usually *u̯eh1 (vgl. ved. (RV 6.55) vā́m usw.), whereas "Acc. 2nd Du." is *u̯oh1 (> aks. va :: ved. =vām usw.). These stems are so similar ... I think there are too many flaws in the traditional reconstruction. This is mainly due to a backprojection of the given (case) categories (which seem to be post-PIE). I think the personal pronouns cannot be reasonably considered a good basis for any cladistic in their presently given forms ... If they are not excluded, but included, these cognate sets make the whole story more uncertain.

**Comment by cormacanderson** [comment]
I agree that as we go through some of the comparison meanings, the problems with establishing cognacy appear so large that they should maybe be entirely excluded from the export. The deictics are primary candidates for exclusion on these grounds.

**Comment by CasFre** [comment]
Okay, so then we leave them for now and concentrate on other things?
The place to discuss this then might be https://github.com/lingdb/CoBL/issues/236.
